files/vif-p2p

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+#============================================================================
+# ${XEN_SCRIPT_DIR}/vif-p2p
+#
+# Script for configuring a vif in routed mode, but using P2P configuration
+# on the vif interface.
+#
+# Usage:
+# vif-p2p (add|remove|online|offline)
```

Comment:
`remove` is not implemented

Reply:
It is implemented. It just does nothing additional to what `vif-common.sh` already does. This is analogous to `vif-route`.

Reply:
We might think about handling `remove` like `offline`. But as I understand it, `remove` means that Xen will remove the interface, so there's not point in setting it down first.